fix(functions): update reviewed namespace creation doc (#1707)
scaleway/docs-content: serverless/functions/how-to/create-a-functions-namespace.mdx

</Message>

1. Click **Functions** in the **Serverless** section of the side menu. The functions page displays.
2. Click **Create namespace**. The functions namespace creation wizard displays.

> 3. Complete namespace informations:

Reviewer: ```suggestion
3. Complete namespace information:
```

    - **Name** of your namepace. The first character of your namespace must be a letter. It can only contain letters, numbers, and dashes. No other special characters are permitted. The namespace is case-sensitive and cannot be edited after its creation.
    - **Description** (optional).
    - **Region** is the geographical location in which your namespace will be deployed.
    - **Environment variables** (optional) are key/value pairs that will be available in your functions as environment variables.
    - **Secrets** (optional) are environment variables which are injected into your function and stored securely, but not displayed in the console after initial validation.
4. Click **Create namespace** to finish.

> Your namespace is now created and you will be redirected to **Functions** tab.

Reviewer: ```suggestion
Your namespace is now created and you are redirected to the **Functions** tab.
```
